### DISCUSSION TITLE
Requires a Tour guide like Android that appears when entering the desktop for the first time after installation

### DISCUSSION THREAD

**Discussion by louies0623** [helloSystem/hello#433] [Ideas]
**android**
![kitkat_launcher](https://user-images.githubusercontent.com/44593430/206849322-07688287-decb-486c-b7b7-b3dcaafadc15.png)
![home-welcome-folder](https://user-images.githubusercontent.com/44593430/206849324-9afb68a7-bfca-41a7-be80-19a108762f86.jpeg)

**ubuntu touch (another qt project)**
I don't have a phone to use this.
https://youtu.be/---qqgT5Swo?t=70

**Comment by probonopd** [comment]
Most people find these annoying. Plus, our target audience already knows the desktop metaphor since 1984.

**Reply by louies0623** [reply]
Yes, but it's better to let first-time users know where the window menu is.

**Reply by probonopd** [reply]
Who does not know where the Menu is should probably better watch this first:
https://www.youtube.com/watch?v=hAilgGS-JHs
